File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter05/CSHARP/BreakSentence.cs
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 05.-Flujo de control II: Estructuras repetitivas
    Problema 09.-Validar un numero de identificacion. 
    Pagina 176
*/
using System;
class BreakSentence{    
    public static void Main(string[] args){

           System.Console.Write("Cuenta del 1 - 25:\n\n");
            for(int i=1; i<=25; i++){
               System.Console.Write(i+" ");

                if(i==5){
                   System.Console.Write("\n\nNumero: "+i+" es hora de salir.\n");
                    break;
                }
            }
    }
}

File: Java2CursoDeProgramacion4aEdicion/javaConsole/chapter07/BreakSentence.java
/**Libro: Java 2 Curso de programación 4a edición
*Capitulo 07: Sentencias de control
*Programa 10: Uso de la sentencia break. 
*/

public class BreakSentence{
    public static void main(String[] args){
        for(int i=1; i<100; i++){
            
            if(i%5!=0){
                break;
            }
            System.out.print(i+" ");
        }
        System.out.println();
    }
}